src/main/java/net/commments/object/BCommentTable.java

```diff
@@ -0,0 +1,22 @@
+package net.commments.object;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class BCommentTable implements CommentTable {
+    private final WebDriver driver;
+
+    public BCommentTable(WebDriver webDriver) {
+        this.driver = webDriver;
+    }
+
+    public void checkCommentInTable(int numberComment) {
+        this.driver.findElements(By.name("SelectedId")).get(numberComment).click();
+    }
+
+
+    public String checkActiveStatus() {
```

Comment:
Do you check something in this method? I don't see. That's why please rename it to `commentStatus` or something similar.

Reply:
done